werkzeug/debug/__init__.py

```diff
@@ -93,7 +93,7 @@ def get_pin_and_cookie_name(app):
     if num is None:
         num = ('%09d' % int(h.hexdigest(), 16))[:9]
 
-    h.update('cookiesalt')
+    h.update('cookiesalt'.encode('utf-8'))
```

Comment:
I didn't dig into what's actually going on there, just thought it might be nice to fix the immediate error (since it will probably be popping up got a lot of people).

Taking a closer look I would say its for "stability" of the code so that the thing does not invalidate after every restart of the server (could be wrong tho...)